### ISSUE TITLE
Unknown rule

### ISSUE THREAD

**Issue by maciej-skurznica** [stylelint/stylelint-config-standard#286]
### What steps are needed to reproduce the bug?

extend from stylelint-config-standard or recommended

### What did you expect to happen?

No errors about unknown rule

### What actually happened?

Every css or styled-component file shows those errors.
See screenshots:
https://drive.google.com/file/d/1GjOlXzjysH23zxrlADymN65gtaMhxMq0/view?usp=share_link
https://drive.google.com/file/d/1o8BdrIs6ypVoptW3JgVAvH_exuLIkcGK/view?usp=share_link
I have browsed through stylelint docs and cannot find the answer. I am using the latest stylelint and sharable configs.

**Comment by maciej-skurznica** [comment]
Is there a way to disable this `Unknown rule` error, or how to fix it?

**Comment by ybiquitous** [comment]
@maciej-skurznica Thanks for the report using the template. Unfortunately, we cannot disable an unknown rule error for now.

We'd like to know your motivation deeper, but we cannot see your screenshots in Google Drive. Instead, can you provide a *minimal* reproducible repository or a [demo](https://stylelint.io/demo/)?

**Comment by maciej-skurznica** [comment]
I have changed permissions for the screenshots so you should be able to see them.

**Comment by maciej-skurznica** [comment]
My motivation: I want to keep using the stylelint but it spoils the experience if I see the 'unknown errors' in every file. Don't you get those errors? I have tried with three repositories and it is the same.

**Comment by ybiquitous** [comment]
Thanks, we can now see the screenshots. But we'd like more info about your situation (Stylelint config, CSS code, dependencies, etc.).

**Comment by maciej-skurznica** [comment]
stylelint:
`
{
"extends": "stylelint-config-standard",
  "overrides": [
    {
      "files": [
        "**/*.{js,jsx,ts,tsx}"
      ],
      "customSyntax": "postcss-styled-syntax"
    }
  ],
}
`

As I mentioned earlier I have been having this issue in three repos. In next.js, react and I even created a clean brand new project and it is the same.

**Comment by maciej-skurznica** [comment]
Can you confirm? Do you see those `Unknow rule` errors in your projects?

**Comment by ybiquitous** [comment]
Unfortunately, I have not used CSS-in-JS so much. 😓 
So, in addition to the Stylelint config, a CSS code snippet (or JSX?) should help us understand your problem.

**Comment by maciej-skurznica** [comment]
Yeah, sure. Screenshots:
https://drive.google.com/file/d/1GIKS8X1C-E7tACB-3yevSvJ95VlMMj0Q/view?usp=sharing
https://drive.google.com/file/d/1JYYZEpgrKghxVTPkZlHbuCIxcd4K2Jmn/view?usp=sharing
https://drive.google.com/file/d/1bw8dzQJ1jLTa3UJ49JSNDjhhJRWgOxy7/view?usp=sharing
https://drive.google.com/file/d/1fRwp0ZoprEOuXvzVTzXg9lezm0zJofIA/view?usp=sharing
https://drive.google.com/file/d/1hXubk0wUHo9w6lt9t01bqZ6rwtj0Orkx/view?usp=sharing
https://drive.google.com/file/d/1mqQfxlxVYbfOVG1V55ogZKIvi1O-ffXJ/view?usp=sharing

**Comment by ybiquitous** [comment]
I can see the screenshots, but I cannot reproduce the errors with CLI. Here is an example that I tried to reproduce the problem:

https://stackblitz.com/edit/node-9yygpm?file=index.jsx

<img width="449" alt="Screenshot 2023-04-09 at 22 01 49" src="https://user-images.githubusercontent.com/473530/230774125-26e50b79-81a5-4f0f-a029-b4adf16bebd7.png">

`package.json`:

```json
{
  "devDependencies": {
    "postcss-styled-syntax": "0.4.0",
    "stylelint": "15.4.0",
    "stylelint-config-standard": "32.0.0",
    "typescript": "5.0.4"
  },
  "scripts": {
    "test": "stylelint \"*.jsx\""
  }
}
```

`.stylelintrc.json`:

```json
{
  "extends": ["stylelint-config-standard"],
  "customSyntax": "postcss-styled-syntax"
}
```

```jsx
import styled from 'styled-components';

export const Container = styled.div`
  color: red;
`;
```

```console
$ npx stylelint "*.jsx" -f verbose

1 source checked
 /home/projects/node-9yygpm/index.jsx

0 problems found
```

The problem may be due to your project's dependencies or editor settings.
